r/R/transforms.R

```diff
@@ -254,7 +254,7 @@ min_by_col <- function(mat, vals) {
   }
   assert_is(mat, "IterableMatrix")
   assert_is(vals, "numeric")
-  assert_greater_than_zero(vals)
+  # assert_greater_than_zero(vals)
```

Comment:
Any zero values would cause this to fail.  Is this intentional?

Reply:
I haven't resolved this, but are you saying that we just need some extra logic to skip a number if zero is iterated on, in `minbycol()`?